summerofcode/2022.md

```diff
@@ -333,3 +333,33 @@ Description: The [in-toto Jenkins plugin](https://github.com/jenkinsci/in-toto-p
 - Recommended Skills: Python
 - Mentors: Lukas Pühringer (@lukpueh)
 - Upstream Issue: https://github.com/theupdateframework/python-tuf/issues/1136
+
+### Keylime
+
+#### Support push model for agent attestation
+
+- Description: [Keylime](http://keylime.dev) enables users to monitor remote nodes (file integrity and measured boot) using a hardware based cryptographic root of trust. Keylime currently operates on a pull basis which means that the tenant or verifier connect to the agent to collect attestation data. This works fine in most virtualized environments where all the devices are in the same network, but not for edge devices or in BYOD contexts. This work would allow remote nodes to work in a "push" model instead of the normal "pull" model.
+- Recommended Skills: Python, Security
+- Mentor(s): Thore Sommer (@THS-on), Michael Peters (@mpeters), Marcio Silva (@maugustosilva)
+- Upstream Issue (URL): https://github.com/keylime/enhancements/issues/60
+
+#### Support push model for agent attestation
```

Comment:
@mpeters Is it intentional that the topic name is identical to the previous one?

Reply:
Good catch. New PR to fix it